README.rst

```diff
@@ -1,6 +1,22 @@
 ftw.referencewidget
-==============
+===================
 
+This Widget provides a Referencebrowser and a searchfield, which allows users to select references.
+
+The basequery is all types which are not in the types_not_searched property all modification to the allowed types are relative to this query.
+
+Traversal or Selectability can be changed for all widgets with the IReferenceSettings registry interface or per widget with the widget parameters.
+
+The widget takes the following parameters:
+ - allow_traversal: These Types are traversable
+ - block_traversal: These Types are not traversable
+ - selectable: These Types are selectable
+ - nonselectable: These Types are not selectable
+ - start: The path first opened. Can either be a callable or a string
```

Comment:
@jone atm you can't although here is default if nothing is given. the question is if we want to hardcode these or if we want to be able to register these if the same are used frequently.